src/markdown/handbook/tools/figma/pocesses.md

```diff
@@ -0,0 +1,85 @@
+---
+title: 'Figma'
+description: 'Processes'
```

Comment:
the description field should [have more details for SEO purposes](https://moz.com/learn/seo/meta-description#:~:text=A%20meta%20description%20is%20an,relates%20to%20their%20search%20query.)